content/departments/product-engineering/engineering/enablement/dev-experience/newsletter.md

```diff
@@ -8,6 +8,135 @@ To learn more about components of Sourcegraph's developer experience, check out
 
 > NOTE: For authors, refer to [this guide](./index.md#newsletter) for preparing a newsletter.
 
+## Feb 21, 2022
+
+Welcome to another iteration of the [Developer Experience newsletter](./newsletter.md) of notable changes since the Jan 10th issue!
+As a reminder, you can check out previous iterations of the newsletter in the [newsletter archive](./newsletter.md).
+
+### SOC2 compliance processes
+
+TODO pr-auditor, test plans
+
+https://github.com/sourcegraph/sourcegraph/pull/30278
+
+https://docs.sourcegraph.com/dev/background-information/testing_principles
+
+https://sourcegraph.com/notebooks/Tm90ZWJvb2s6NjA=
+
+### Internal tools and libraries
+
+#### Database migrations update
+
+TODO ask eric
+
+#### New `lib/errors` package
+
+_All_ errors in Sourcegraph backend services should now use the new `github.com/sourcegraph/sourcegraph/internals/errors` package. This consolidation helps us restrict and control the ways that we can create, consume, and compare errors, and will allows us to control library behavior clashes more easily in the future. [#30558](https://github.com/sourcegraph/sourcegraph/pull/30558)
+
+#### Actor propagation reminder
+
+Unified actor propagation was introduced a few months ago as part of an effort to enable the implementation of [sub-repository permissions](https://github.com/sourcegraph/sourcegraph/issues/27916) across all Sourcegraph features.
+There have been gradual efforts to roll out this actor propagation to more services, which may cause behavioural changes that impact how permissions are handled if, for example, `internal` actors are not set explicitly.
+When implementing new features please ensure that actors are correctly set and read from contexts.
+
+To learn more, check out the [intro to actor propagation search notebook](https://sourcegraph.com/notebooks/Tm90ZWJvb2s6OTI=).
```

Comment:
cc @mollylogue  let me know if there's anything you want to add here, since you've been working on investigating missing actors!